### PR TITLE
Fix #22 The steps must be displayed in the same order as executed

### LINKED ISSUE #22: The steps (@Test, the first level) must be displayed in the same order as executed
The steps (@Test, the first level) must be alphabetically ordered inside each Test Case (@tmslink)
It would be better to also order alphabetically the Test Cases inside a suite.

### PR DESCRIPTION
**Pull request by olgapuig**

### CODE REVIEW

**Comment by olgapuig** [comment]
Fix the following issues:
- #22 The steps must be displayed in the same order as executed
- #27 Show additional labels on the report such as version, tester, execution date, browser
- #32 In Angular, an it without step is not documented
- #33 TestCase is displayed as Failed when a step is disabled with xit

File: src/app/app.component.ts [hunk 1]
```diff
@@ -86,6 +86,7 @@ export class AppComponent {
 
 					// Step number must be incremental after the sorting
 					this.testSuites.forEach((suite) => {
+						suite.testCases.sort((a, b) => a.start < b.start ? -1 : a.start < b.start ? 1 : 0);
```
Comment (oGalceran1) [inline src/app/app.component.ts]: Are you sure about this line of code?

Comment (olgapuig) [reply src/app/app.component.ts]: Yes, this line is the fixing for the issue #22. It's required to sort the steps in the same order as executed.

Comment (oGalceran1) [reply src/app/app.component.ts]: The reason is clear, but the code is correct or it should be:
suite.testCases.sort((a, b) => a.start < b.start ? -1 : a.start > b.start ? 1 : 0);
I mean change the second **a.start < b.start** by **a.start > b.start** ???

Comment (olgapuig) [reply src/app/app.component.ts]: You're right. I've just uploaded the fixing.

**Pull request merged**